build/package/Installer.RPM.targets

```diff
@@ -68,13 +68,13 @@
     <!-- Replace config json variables -->
     <ItemGroup>
       <SDKTokenValue Include="%SHARED_HOST_RPM_VERSION%">
-        <ReplacementString>$(MicrosoftNETCoreAppPackageVersion)</ReplacementString>
+        <ReplacementString>$(MicrosoftNETCoreAppMajorMinorVersion)</ReplacementString>
       </SDKTokenValue>
       <SDKTokenValue Include="%ASPNETCORE_SHAREDFX_RPM_PACKAGE_NAME%">
         <ReplacementString>$(AspNetCoreSharedFxRpmPackageName)</ReplacementString>
       </SDKTokenValue>
       <SDKTokenValue Include="%ASPNETCORE_SHAREDFX_RPM_PACKAGE_VERSION%">
-        <ReplacementString>$(AspNetCoreVersion)</ReplacementString>
+        <ReplacementString>$(AspNetCoreMajorMinorVersion)</ReplacementString>
```

Comment:
This is wrong. The package name is aspnetcore-runtime-2.1 but the version is 2.1.0 for this release. Future patches will be 2.1.1, etc.